packages/app/src/sandbox/eval/utils/resolve-utils.js

```diff
@@ -1,6 +1,5 @@
-export function packageFilter(p: Object) {
```

Comment:
@CompuIves How was this even working?! 🙂

Reply:
Well, how was it not working? Or do you mean the type definitions?

Reply:
> Or do you mean the type definitions?

Yes, looks like that was left from the `flow` days?